src/Package/Impl/Help/HelpVisualComponent.cs

```diff
@@ -266,6 +266,9 @@ internal sealed class HelpVisualComponent : IHelpVisualComponent {
 
         private static bool IsHelpUrl(string url) {
             Uri uri = new Uri(url);
+            if(uri.AbsoluteUri.EndsWithIgnoreCase(".pdf")) {
```

Comment:
> No, still should open PDF in external.
I'm not sure if such check is reliable for external url, which may have no file name in it at all.